rails-install-ubuntu.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
-curl -sL https://deb.nodesource.com/setup_dev | sudo bash -
```

Comment:
The curl dependency will remain, as it's already required to fetch the installation script.
See https://github.com/railsgirls/installation-scripts/commit/958ac0542f4af7841dce05e752c764cd50a970f5#diff-04c6e90faac2675aa89e2176d2eec7d8R11